aspnet/fundamentals/dependency-injection.rst

```diff
@@ -1,10 +1,277 @@
-.. _dependency-injection:
+Dependency Injection
+====================
+By `Steve Smith`_
 
-.. include:: /../common/stub-topic.txt
+ASP.NET 5 is designed from the ground up to support and leverage dependency injection. ASP.NET 5 applications can leverage built-in framework services by having them injected into methods in the Startup class, and application services can be configured for injection as well. If desired, developers can replace the built-in services container with a custom or third party implementation.
```

Comment:
> having them injected into methods in the Startup class

Suggest "injecting them into application classes". The `Startup` class is a special case.